pkg/basetypes.go

```diff
@@ -45,12 +46,13 @@ const (
 	PerformEvent TransmitEventType = iota
 	StaleReportEvent
 	ReorgReportEvent
+	InsufficientFundsReportEvent
```

Comment:
This event is emitted in the transmit function on the contract? I must have missed that one.

Reply:
Yes, that is from the contract.